.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+###### Description
+
+
+###### System Info
+macOS 10.
+Xcode 8.
+
+###### Verification
+- [ ] Have you checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
+- [ ] Have you referenced relating tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?
+- [ ] Have you checked your Portfile with `port lint`?
+- [ ] Have you tried existing tests with `sudo port test`?
+- [ ] Have you tried a full install with `sudo port install`?
+- [ ] Have you tested basic functionality of all binary files?
```

Comment:
Yes, rev-bumping is only needed when libraries with api changes are involved. So it's not all ports, but for some ports it's super painful (like Qt) and some developers completely forget about this (I would say "me included" because most of the time I'm not dealing with such ports).